src/config.ts

```diff
@@ -1,3 +1,10 @@
 export const Config = {
-  port: parseInt(process.env.PORT || "8080"),
+  port: parseInt(process.env.PORT || "3000"),
+  secretUrl: "/e775veinteTwo",
```

Comment:
What do you think about changing this?

```suggestion
  secretUrl: process.env.SECRET_URL || "/SECRET",
```

Reply:
(remember that this repository is public)